src/tools/annotation/EllipticalRoiTool.js

```diff
@@ -49,9 +49,11 @@ export default class EllipticalRoiTool extends BaseAnnotationTool {
       configuration: {
         // showMinMax: false,
         // showHounsfieldUnits: true,
+        drawEllipseHandles: false,
```

Comment:
This should be named `drawHandles` and be defaulted to `true` to be consistent with the other ROI tools. See https://github.com/videahealth/cornerstoneTools/blob/master/src/tools/annotation/RectangleRoiTool.js#L45